src/System.Runtime.CompilerServices.Unsafe/src/System.Runtime.CompilerServices.Unsafe.il

```diff
@@ -0,0 +1,158 @@
+  .custom instance void [System.Runtime]System.Reflection.AssemblyProductAttribute::.ctor(string) = ( 01 00 1A 4D 69 63 72 6F 73 6F 66 74 C2 AE 20 2E   // ...Microsoft.. .
+                                                                                      4E 45 54 20 46 72 61 6D 65 77 6F 72 6B 00 00 )    // NET Framework..
+  .custom instance void [System.Runtime]System.CLSCompliantAttribute::.ctor(bool) = (
+    01 00 00 00 00
+  ) // false
+  .hash algorithm 0x00008004
+  .ver 4:0:0:0
+}
+.module System.Runtime.CompilerServices.Unsafe.dll
+// MVID: {1E97D84A-565B-49C5-B60A-F31A1A4ACE13}
+.imagebase 0x00400000
+.file alignment 0x00000200
+.stackreserve 0x00100000
+.subsystem 0x0003       // WINDOWS_CUI
+.corflags 0x00000001    //  ILONLY
+// Image base: 0x02ED0000
+
+
+// =============== CLASS MEMBERS DECLARATION ===================
+
+.class public abstract auto ansi sealed beforefieldinit System.Runtime.CompilerServices.Unsafe
+       extends [System.Runtime]System.Object
+{
+  .method public hidebysig static !!T  Read<T>(void* p) cil managed aggressiveinlining
+  {
+        .custom instance void System.Runtime.Versioning.NonVersionableAttribute::.ctor() = ( 01 00 00 00 )
+        .maxstack  8
+        ldarg.0
+        ldobj !!T
+        ret
+  } // end of method Unsafe::Read
+
+  .method public hidebysig static void  Write<T>(void* p,
+                                                 !!T 'value') cil managed aggressiveinlining
+  {
+        .custom instance void System.Runtime.Versioning.NonVersionableAttribute::.ctor() = ( 01 00 00 00 )
+        .maxstack  8
+        ldarg.0
+        ldarg.1
+        stobj !!T
+        ret
+  } // end of method Unsafe::Write
+
+  .method public hidebysig static void  Write<T>(void* p,
+                                                 !!T& 'value') cil managed aggressiveinlining
+  {
+        .custom instance void System.Runtime.Versioning.NonVersionableAttribute::.ctor() = ( 01 00 00 00 )
+        .maxstack  8
+        ldarg.0
+        ldarg.1
+        ldobj !!T
+        stobj !!T
+        ret
+  } // end of method Unsafe::Write
+
+  .method public hidebysig static void*  AsPointer<T>(!!T& 'value') cil managed aggressiveinlining
+  {
+        .custom instance void System.Runtime.Versioning.NonVersionableAttribute::.ctor() = ( 01 00 00 00 )
+        .maxstack  8
+        ldarg.0
+        conv.u
+        ret
+  } // end of method Unsafe::AsPointer
+
+  .method public hidebysig static int32  SizeOf<T>() cil managed aggressiveinlining
+  {
+        .custom instance void System.Runtime.Versioning.NonVersionableAttribute::.ctor() = ( 01 00 00 00 )
+        .maxstack  8
+        sizeof !!T
+        ret
+  } // end of method Unsafe::SizeOf
+
+  .method public hidebysig static void CopyBlock(void* destination, void* source, uint32 numBytesToCopy) cil managed aggressiveinlining
```

Comment:
The framework convention for Copy methods is to take source first and destination second (`Array.Copy`, `Buffer.BlockCopy`, ...). Do we want to follow it here as well?